lib/ReactViews/Search/SearchBoxAndResults.tsx

```diff
@@ -67,85 +73,65 @@ const PresentationBox = styled(Box).attrs({
 
 export const LOCATION_SEARCH_INPUT_NAME = "LocationSearchInput";
 
-export class SearchBoxAndResultsRaw extends React.Component {
-  constructor(props) {
-    super(props);
-    this.locationSearchRef = React.createRef();
-  }
+interface SearchBoxAndResultsProps {
+  placeholder?: string;
```

Comment:
The placeholder is set as `isRequired` in the old code, is the `?` intentional here?

Reply:
It is only used in the `SearchBox`, and it is not required there; furthermore, tests did not provide the placeholder value, so I assumed it is safe to make it optional (can make it required if needed).

Reply:
I have made it required again